app/assets/javascripts/awesome_react.js

```diff
@@ -0,0 +1,40 @@
+var CommentStore = {
+  items: [],
+  subscribers: [],
+
+  subscribe: function(parent, callback) {
+    this.subscribers.push({ parent: parent, callback: callback });
+  },
+
+  unsubscribe: function(parent, callback) {
+    this.subscribers = this.subscribers.filter(subscriber => (subscriber.parent == parent && subscriber.callback == callback));
```

Comment:
Assuming every subscriber is bound like I mentioned in the other comment, it should be enough to check `subscriber.callback === callback` here, since each bound callback will be unique to the instance